website/docs/reference/global-configs/warnings.md

```diff
@@ -4,7 +4,15 @@ id: "warnings"
 sidebar: "Warnings"
 ---
 
-Turning on the `WARN_ERROR` config will convert dbt warnings into errors. Any time dbt would normally warn, it will instead raise an error. Examples include `--select` criteria that selects no resources, deprecations, configurations with no associated models, invalid test configurations, or tests and freshness checks that are configured to return warnings.
+Enabling `WARN_ERROR`config or setting the `--warn-error` flag will convert dbt warnings into errors. Any time dbt would normally warn, it will instead raise an error. Examples include `--select` criteria that selects no resources, deprecations, configurations with no associated models, invalid test configurations, or tests and freshness checks that are configured to return warnings.
+
+ You would commonly use the `--warn-error` flag to promote test warnings from `severity: warn` to errors, but it actually affects all warning types, including:
```

Comment:
would we normally just use this for test warnings? for some reason, i thought it could also be used in general for all warnings since its a flag you can pass as a command. If so, i'm not sure we should say that's its used generally to promote test warnings to errors 

```suggestion
You can use the `--warn-error` flag to promote all warnings to errors, such as:
```